src/context/authContext.js

```diff
@@ -22,7 +22,6 @@ export const AuthProvider = ({ children }) => {
             }));
             if (userResponse.statusText === "Created") {
```

Comment:
Always, try to Check the API response with statusCode or any boolean values.

Reply:
Done.